custom_components/tuya_local/devices/duux_blizzard_portable_aircon.yaml

```diff
@@ -142,3 +152,11 @@ secondary_entities:
             value: Celsius
           - dps_val: true
             value: Fahrenheit
+  - entity: sensor
+    name: countdown timer
```

Comment:
The first word of entity names should be capitalised according to Home Assistant conventions.  The usual name for this is "Time remaining", or just "Timer" as it is linked to the number Timer above.